File: service/Ebbinghaus.Domain/Word/Sentence.cs

using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

[BsonIgnoreExtraElements]
public class Sentence
{
    public Guid id { get; set; }
    public string text { get; set; }
    public DateTime created { get; set; }
    public DateTime? updated { get; set; }

    public Sentence(string text)
    {
        this.id = Guid.NewGuid();
        this.text = text;
        created = DateTime.Now;
    }
}

File: service/Ebbinghaus.Job/models/sentence.go
package models

import (
	"time"
)

type Sentence struct {
	id    string  `bson:"_id,omitempty"`
	text string  `bson:"text,omitempty"`
	created time.Time  `bson:"created,omitempty"`
	updated time.Time  `bson:"updated,omitempty"`
 }